lib/streamlit/runtime/scriptrunner/script_run_context.py

```diff
@@ -107,6 +111,12 @@ def enqueue(self, msg: ForwardMsg) -> None:
         # Pass the message up to our associated ScriptRunner.
         self._enqueue(msg)
 
+    def check_both_query_apis_called(self):
```

Comment:
This name is awkward because it sounds like we want both to be used rather than it being something that's not allowed